graph_nets/tests/modules_test.py

```diff
@@ -131,15 +144,27 @@ def _get_model(self, name=None):
       kwargs["name"] = name
     return modules.GraphIndependent(**kwargs)
 
-  def test_same_as_subblocks(self):
+  def _get_model_w_norm(self):
+    kwargs = {
+        "edge_model_fn": functools.partial(ModelWithNorm, output_sizes=[5]),
+        "node_model_fn": functools.partial(ModelWithNorm, output_sizes=[10]),
+        "global_model_fn": functools.partial(ModelWithNorm, output_sizes=[15]),
+    }
+    return modules.GraphIndependent(**kwargs)
+
+  @parameterized.named_parameters(
+      ("with scale and offset", {"scale": 2, "offset": 1},
+       {"scale": .5, "offset": .25}, {"scale": 3, "offset": 1.5})
+  )
+  def test_same_as_subblocks(self, edge_kw, node_kw, global_kw):
```

Comment:
It would be great it this functionality could be tested without modifying the existing tests (so we know we are not breaking any previous users).

Would it be possible to instead add a single test for each module using the same idea that I described in the `blocks_test`. E.g. compare that the function output is the same whether the parameter passed via an arg to the build, or hardcoded in the model_fn:

e.g.

compare
```
edge_model_fn = functools.partial(ModelWithNorm, output_sizes=[5])
graph_network = GraphNetwork(edge_model_fn,...)
graph_network(input, scale=5, offset=3)

```

to:

```
edge_model_fn = functools.partial(ModelWithNorm, output_sizes=[5])
edge_model_fn_with_params = lambda: functools.partial(edge_model_fn(), scale=5, offset=3)
graph_network = GraphNetwork(edge_model_fn_with_params,...)
graph_network(input)
```

(you will need to remove the MLP part from ModelWithNorm so you don't get different weights, and also this test may actually fail in Sonnet2 if ModelWithNorm has variables, as it may struggle to find them hidden in the partial func.